cartography/intel/aws/apigateway.py

```diff
@@ -35,6 +37,8 @@ def get_client_certificates(boto3_session: boto3.session.Session, region: str, c
             certificates.extend(page['items'])
         for certificate in certificates:
```

Comment:
we should move the transformation logic to a different function. this `get_client_certificates` function should focus only on getting data. Transform function can do the enrichment of attributes like consolelink, arn, etc.